Drop note about Pi 3 2.4GHz only WiFi

scop opened the pull request:
**Description:**

The (currently unsupported) B+ has 5GHz support.

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

scop committed:
getting-started: Soften note about Pi 3 being 2.4GHz WiFi only

The (currently unsupported) B+ has 5GHz support.

fabaff commented on source/getting-started/index.markdown:
```diff
@@ -41,7 +41,7 @@ We will need a few things to get started with installing Home Assistant. Links b
 
 1. Put the SD card in your SD card reader.
 1. Open Etcher, select the Hass.io image and flash it to the SD card.
-1. WiFi setup only: open the file `system-connections/resin-sample` (from the `resin-boot` volume on the SD card) with a text editor. Change `ssid` to be your network name and `psk` to be your password. Note that the Raspberry Pi 3 is a 2.4GHz WiFi device, so do not try to connect it to a 5GHz network.
```
The note is not about the Raspberry Pi B+. Thus, it's still valid.

scop replied:
The note is about "Raspberry Pi 3", which to me includes all Pi 3 models, including B and B+.

NovapaX replied:
ok. Then it should be changed to something 'Raspberry Pi 3B (standard version, not the +-version)'

scop replied:
I think that's too much detail, and that just removing the note would be a better idea.

NovapaX replied:
Then I think we should just replace it with:
`Note: When connecting to a 5Ghz-only network, make sure your device supports that. If you don't know what that means, then you probably have nothing to worry about.`

scop replied:
WFM

scop replied:
Just to make it clear, because @NovapaX came up with the improved wording, I think it would be a good idea for them to submit a superseding PR to get the authorship logged right.

frenck replied:
Feel free to take make the change @scop.
Members of the Home Assistant project (like me, @fabaff and @NovapaX) make suggestions on a daily basis and are not seeking for the credit on every single change/PR 😉

fezfox committed:
Confusing as it uses a non-default TTS platform

fezfox committed:
Update sonos_say.markdown

sveip committed:
Update debugging.markdown

Hassio supervisor container name corrected.

DubhAd committed:
Merge pull request #5075 from fezfox/patch-1

Confusing as it uses a non-default TTS platform

DubhAd committed:
Merge pull request #5076 from sveip/patch-2

Update debugging.markdown

dale3h committed:
Add quote explanation and examples

JohnnyCAPSLOCK committed:
Update google_assistant.markdown

Proposing changes to steps 8a and 8d.  I looked back at previous versions and I don't think these steps were updated when the other steps it was originally referencing were moved/changed.

DubhAd committed:
Merge pull request #5081 from dale3h/quote-examples

Add quote explanation and examples

DubhAd committed:
Merge pull request #5082 from JohnnyCAPSLOCK/patch-2

Update google_assistant.markdown

fabaff committed:
Remove not needed raw's

VdkaShaker committed:
Update configure filter section (#5071)

Adding detail to Configure Filter section.

cdce8p committed:
Added upgrade note (#5087)

* Added upgrade note

* Minor changes

DubhAd committed:
Added note about broadcast group (#5080)

Thanks to turbokongen commenting in https://github.com/home-assistant/home-assistant/issues/11823

Added a note about how to remove the broadcast group association

sebk-666 committed:
Update hue.markdown

- Removed the "(Optional)" tag from the host-Parameter, because it is mandatory now
- Added another configuration example to clarify how to specify multiple configuration variables; the examples in the "multiple bridges..." section do show this already, but this section might be skipped by the majority of users owning only a single bridge.

frenck reviewed (changes requested):
Awaiting change.

ChristianKuehnel committed:
bmw_connected_drive - updated repo url (#5044)

* updated repo url

* :ambulance: Fixes incorrect Markdown linking

DubhAd committed:
Merge pull request #5089 from sebk-666/patch-6

Update hue.markdown

fabaff committed:
Some fixes (#5091)

* Some fixes (missing logo and links)

* Add header

JudgeDreddKLC committed:
minor grammar changes to improve readability (#5092)

minor grammar changes to improve readability

worm-ee committed:
Rename and fix naming in homematicip_cloud (#5088)

* Rename homematicip to homematicip_cloud

* Fix homematicip_cloud naming

* Update link and formating

Commit:
Added title field into example (#5095)

Added based on feedback from @Stringyb92 in Chat.  Examples were unclear/inconsistent.

fabaff committed:
Add more details

343max committed:
Update troubleshooting.markdown (#5097)

* Update troubleshooting.markdown

By default the check_config script only search `~/.homeassistent` for a configuration.

* Fix typo

* Add link

evemorgen committed:
Extend development 101 (#5099)

* extend development 101 with configuration.yaml entry and  runtime example.

* being more descriptive, change quote marks to follow standards

* Minor changes

Skaronator committed:
Match code (again) (#5106)

Pengman committed:
Fixed link to target sensor.mqtt (#5104)

cgarwood committed:
Remove reference to old_entity_id and new_entity_id attributes (#5110)

Remove references to old_entity_id and new_entity_id attributes as they were removed in https://github.com/home-assistant/home-assistant/pull/12652

filcole committed:
Update Debian Stabel python version

DubhAd committed:
Merge pull request #5111 from filcole/patch-5

Update Debian stable python version

leofuscaldi committed:
Update camera.yi.markdown (#5098)

* Update camera.yi.markdown

Added a session specifying for Hassbian users.

* Update camera.yi.markdown

* :rocket: Trigger build

jcs committed:
Pushover: update webpage with clone URL (#5116)

* Pushover: update webpage with clone URL

Hi, Pushover developer here.

I was notified that your app has a Pushover plugins, so I've listed it on our [apps page](https://pushover.net/apps) and enabled cloning for it.  Instead of users having to upload an icon and fill in all the fields, users can be directed to https://pushover.net/apps/clone/home_assistant which pre-fills those fields and copies the icon.

I'm not sure if the `date` field in this markdown page needs to be bumped when it's updated, or if it's a creation date.

* Add some other minor changes

VoidLock committed:
add supporting info for 2nd gen august lock (#5115)

* add supporting info for 2nd gen august lock

2nd Gen August Lock needs to have the August Connect Module to be able to talk to home-assistant. If you have Doorbell then you don't need to have Connect. Just trying to help people troubleshoot in case they don't know whats going wrong.

* Update syntax

syssi committed:
Ceiling Light 4 (Jiaoyue 650) added to the list of supported devices (#5119)

scop commented:
Hmm, this PR is incorrectly requesting merge to next, current would be the intention. I'll close this and open a new one, as it seems it's not possible to change the merge target afterwards.

NovapaX commented:
For future reference: https://blog.github.com/2016-08-15-change-the-base-branch-of-a-pull-request/
